class-tour.php

```diff
@@ -52,15 +52,12 @@ public static function enqueue_scripts() {
 			$_tours,
 			function ( $tour ) {
 				$restrict_url = $tour[0]['tour_restrict_url'];
-				$url_pattern  = '#/wp-admin/post\.php\?post=#';
-				if ( empty( $tour[0]['tour_restrict_url'] ) || preg_match( $url_pattern, $_SERVER['REQUEST_URI'] ) ) {
+				$create_tour_url_pattern  = '#/wp-admin/post\.php\?post=#';
+				if ( empty( $tour[0]['tour_restrict_url'] ) || preg_match( $create_tour_url_pattern, $_SERVER['REQUEST_URI'] ) ) {
```

Comment:
Could we make this a non preg_match and add the actual post_id so that it's just always enabled for the current tour?